app/main.py

```diff
@@ -1,3 +1,61 @@
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: [int, float]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: [int, float]) -> [int, float]:
```

Comment:
I meant `Distance` without `"`. 
```suggestion
    def __add__(self, other: Distance) -> Distance:
```

Reply:
yes, but when I do without " I get an error:
app\main.py:12: in Distance
    def __add__(self, other: Distance) -> Distance:
E   NameError: name 'Distance' is not defined
=============================================== short test summary info ================================================ 
ERROR tests/test_main.py - NameError: name 'Distance' is not defined

Reply:
Yeah, sorry, my bad. You can change back to `"`. Sorry for that. Change back to what it was before and I will approve it.

Reply:
no problem)